R/map_metadata_convert.R

```diff
@@ -8,7 +8,23 @@
 #' @return The function will return 'L-OUTPUT_' in the same output_dir
 #' @export
 #' @importFrom utils read.csv write.csv
-
+#' @examples
```

Comment:
This looks good.  If easier, we could decide to include an example output from map_metadata in this location: https://github.com/aim-rsf/browseMetadata/tree/main/inst/outputs (just as we have included for browse_metadata). Then we could load that example data in, rather than having to create and save a csv in the example. What do you think?

Reply:
In fact, if we included *two* example output CSVs from `map_metadata`, and that would help with the example for `map_metadata_compare` also

Reply:
If these example outputs would be helpful, I can create them, then you can implement them into the examples.

Reply:
Yessssss, that's would be great - please do create them and I will implement them.

Reply:
Okay great - let me create them. I think that it will make it much easier for you to write these examples. Because the point of map_metadata_convert and `map_metadata_compare` is that they are run _after_ `map_metadata`.

Reply:
Pushed the example files to this branch! Should be easier to write simple `@example` sections now because you don't need to write the code that creates them 

This is the syntax to locate them I think

`output_demo <- system.file("outputs", "filename.csv", package = "browseMetadata")`

Reply:
<img src="https://media3.giphy.com/media/fem4oy4LOfbL4G64ie/giphy.gif"/>